src/main/java/bio/terra/service/load/Load.java

```diff
@@ -1,58 +1,22 @@
 package bio.terra.service.load;
 
+import java.time.Instant;
+import java.util.Objects;
 import java.util.UUID;
-import org.apache.commons.lang3.builder.ToStringBuilder;
-import org.apache.commons.lang3.builder.ToStringStyle;
 
-public class Load {
-  private UUID id;
-  private String loadTag;
-  private boolean locked;
-  private String lockingFlightId;
-
-  public UUID getId() {
-    return id;
-  }
-
-  public Load id(UUID id) {
-    this.id = id;
-    return this;
-  }
-
-  public String getLoadTag() {
-    return loadTag;
-  }
-
-  public Load loadTag(String loadTag) {
-    this.loadTag = loadTag;
-    return this;
-  }
-
-  public boolean isLocked() {
-    return locked;
-  }
-
-  public Load locked(boolean locked) {
-    this.locked = locked;
-    return this;
-  }
-
-  public String getLockingFlightId() {
-    return lockingFlightId;
-  }
-
-  public Load lockingFlightId(String lockingFlightId) {
-    this.lockingFlightId = lockingFlightId;
-    return this;
-  }
-
-  @Override
-  public String toString() {
-    return new ToStringBuilder(this, ToStringStyle.JSON_STYLE)
-        .append("id", id)
-        .append("loadTag", loadTag)
-        .append("locked", locked)
-        .append("lockingFlightId", lockingFlightId)
-        .toString();
+public record Load(
+    UUID id,
+    String loadTag,
+    boolean locked,
+    String lockingFlightId,
+    UUID datasetId,
+    Instant createdDate) {
+
+  /**
+   * @return whether this Load is locked by the supplied flight ID and dataset
+   */
+  public boolean lockedBy(String lockingFlightId, UUID datasetId) {
+    return Objects.equals(this.lockingFlightId, lockingFlightId)
+        && Objects.equals(this.datasetId, datasetId);
```

Comment:
Is it possible for `lockingFlightId` or `datasetId` to be `null`? If not, this is the same as calling `equals()`.
```suggestion
    return lockingFlightId.equals(lockingFlightId, lockingFlightId)
        && datasetId.equals(datasetId, datasetId);
```

Using `equals()` is slightly shorter and makes it clearer that `null`s are not expected here.